plugins/modules/mysql_user.py

```diff
@@ -22,7 +22,8 @@
     required: true
   password:
     description:
-      - Set the user's password.
+      - Set the user's password. Only for C(mysql_native_password) authentication. 
```

Comment:
```suggestion
      - Set the user's password. Only for C(mysql_native_password) authentication.
```

Reply:
This trailing spaces is a real pain :-/ In the Email you can't even see this one. fixed.

Reply:
@hubiongithub yep:) BTW for future contributions, you can run sanity and other tests locally before pushing. See https://docs.ansible.com/ansible/devel/community/create_pr_quick_start.html (Prepare you environment and Test your changes chapters). Can save time.